shared/teams/team/invites/invite-item.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import * as Container from '../../../util/container'
+import * as Types from '../../../constants/types/teams'
+import * as TeamsGen from '../../../actions/teams-gen'
+
+export type Mode = 'header' | 'settings' | 'history' | 'generate'
```

Comment:
I think we should ditch this component knowing about where it's used in favor of this taking a `style` and prop to hide the "created by" line